protoc-gen-twirp/generator.go

```diff
@@ -243,6 +245,14 @@ func (t *twirp) generate(file *descriptor.FileDescriptorProto) *plugin.CodeGener
 	return resp
 }
 
+func (t *twirp) generateVersionCheck(file *descriptor.FileDescriptorProto) {
+	t.P(`// This is a compile-time assertion to ensure that this generated file`)
+	t.P(`// is compatible with the twirp package used in your project.`)
+	t.P(`// A compilation error at this line likely means your copy of the`)
+	t.P(`// twirp package needs to be updated.`)
+	t.P(`const _ = `, t.pkgs["twirp"], `.TwirpPackageIsVersion7`)
```

Comment:
This seems very similar to the protobuf assertion line, so I think this is a good idea to add so we can also validate the twirp version. LGTM!